Devscord.DiscordFramework/ControllerInfo.cs

```diff
@@ -1,24 +1,30 @@
 ﻿using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Framework.Commands;
+using Devscord.DiscordFramework.Middlewares.Contexts;
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
 
 namespace Devscord.DiscordFramework
 {
-    internal class ControllerInfo
+    public class ControllerInfo
     {
         public IController Controller { get; private set; }
-        public IEnumerable<MethodInfo> Methods { get; private set; }
+        public IEnumerable<MethodInfo> MethodsWhichAreCommands { get; private set; }
```

Comment:
`MethodsWhichAreCommands` skomplikowana nazwa + nie widze sensu w nazywaniu tego tak
jeśli coś jest tutaj to wiadome że jest komendą